src/components/StorePage/index.tsx

```diff
@@ -1,17 +1,16 @@
 import React from 'react';
 import Icon from '@ant-design/icons';
 
+import StoreCollectionsContainer from '../../containers/StoreCollections';
+
 import './style.less';
 import { ReactComponent as SwagIcon } from '../../assets/icons/swag-icon.svg';
 
 const StorePage: React.FC = () => {
   return (
     <div className="store-page">
-      <h1>ACM Store</h1>
-      <div className="coming-soon">
-        <Icon className="temp-icon" component={SwagIcon} />
-        <p>Coming Soon!</p>
-      </div>
+      <h1>Diamond Outfitters</h1>
```

Comment:
I wonder if this is could be a dynamic property.

Reply:
Are you talking about displaying a different element depending on the store's status (if the store has nothing for sale/can't fetch collections/under maintenance)?

Reply:
I was more or less referring to the store name changing depending on store name; for instance, maybe we want the store to be called something else after a while. It turns out, however, that this is supposed to be replaced with the store header component anyway, so it doesn't matter.